client-cli/src/command/wallet_command.rs

```diff
@@ -1,11 +1,14 @@
-use client_common::{Error, ErrorKind, Result};
-use client_core::WalletClient;
-use quest::{ask, success};
+use bip39::{Language, Mnemonic};
```

Comment:
should this functionality be here, as it'll be needed in client-rpc + if the client-* is exposed as a library?

Reply:
I don’t think HD wallet code can be moved to `client-common` because it works with other parts in `client-core`. One possible change is to return `SecUtf8` instead of `Mnemonic` when creating a new wallet so that we can remove `bip39` dependency from `client-cli` and `client-rpc`.

Reply:
changing SecUtf8 makes sense